src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -86,9 +89,20 @@ export class PoolListComponent implements OnInit {
         permission: 'delete',
         icon: 'fa-trash-o',
         click: () => this.deletePoolModal(),
-        name: this.actionLabels.DELETE
+        name: this.actionLabels.DELETE,
+        disable: () => !this.selection.first() || !this.monAllowPoolDelete,
+        disableDesc: () => this.getDisableDesc()
       }
     ];
+
+    this.configurationService.get('mon_allow_pool_delete').subscribe((data: any) => {
+      if (_.has(data, 'value')) {
+        const monSection = _.find(data.value, (v) => {
```

Comment:
You could use the [concise body syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Function_body) with the implied "return". 
```suggestion
        const monSection = _.find(data.value, (v) => v.section === 'mon') || { value: false };
```

Reply:
> You could use the [concise body syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Function_body) with the implied "return".

IMO this is a reason why Perl is not used anymore, it's really hard to read if you use all build in tricks.

Reply:
To make it more clear @s0nea, I just thought you have overseen at while refactoring, as you use the short syntax in other places to and you don't have to use it. I already approved this PR so I'm fine with it :)

Reply:
Okay, understood @Devp00l and thanks for the suggestion. I would leave it as it is.